### PR TITLE
[en] Mention kubernetes/websites everywhere in the same way

### PR DESCRIPTION
**Pull request by MaxymVlasov**
Related to https://kubernetes.slack.com/archives/C0191RDKHU1/p1680549820387879

### CODE REVIEW

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Per policy, we don't maintain blogs older than one year. Therefore, I don't recommend amending the blog content. However, the rest of the content LGTM. Are you sure these are the only pages that have the issue, @MaxymVlasov ?

**Comment by MaxymVlasov** [comment]
@divya-mohan0209 ping

**Comment by MaxymVlasov** [comment]
@divya-mohan0209 pong

File: content/en/releases/release.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ type: docs
 auto_generated: true
 ---
 
-<!-- THIS CONTENT IS AUTO-GENERATED via ./scripts/releng/update-release-info.sh in k/website -->
+<!-- THIS CONTENT IS AUTO-GENERATED via ./scripts/releng/update-release-info.sh in kubernetes/website -->
```
Comment (kbhawkey) [inline content/en/releases/release.md:7]: I guess this content is generated by a script?

File: content/en/docs/contribute/_index.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ end
 subgraph second[Review]
 direction TB
    T[ ] -.-
-   D[Look over the<br>K8s/website<br>repository] --- E[Check out the<br>Hugo static site<br>generator]
+   D[Look over the<br>kubernetes/website<br>repository] --- E[Check out the<br>Hugo static site<br>generator]
```
Comment (kbhawkey) [inline content/en/docs/contribute/_index.md:79]: Does this change affect the image?

File: content/en/docs/contribute/_index.md [hunk 3]
```diff
@@ -123,7 +123,7 @@ flowchart LR
     direction TB
     S[ ] -.-
     G[Review PRs from other<br>K8s members] -->
-    A[Check K8s/website<br>issues list for<br>good first PRs] --> B[Open a PR!!]
+    A[Check kubernetes/website<br>issues list for<br>good first PRs] --> B[Open a PR!!]
```
Comment (kbhawkey) [inline content/en/docs/contribute/_index.md:126]: Preview:
https://deploy-preview-40493--kubernetes-io-main-staging.netlify.app/docs/contribute/#your-first-contribution

File: content/en/releases/release.md (hunk 1 above)
Comment (MaxymVlasov) [reply content/en/releases/release.md:7]: Yep, https://github.com/kubernetes/website/pull/40499

**Review by sftim — COMMENTED** [review]
I'd rather write `k/website` everywhere as that's how the community refers to this repo. In some places, we might want to explain what the `k` is short for.

@kbhawkey (you LGTMed this) - which approach do you prefer? My suggestion, or as in this PR as it stands?

**Comment by kbhawkey** [comment]
Hi. My thinking is that we should refer to the Kubernetes docs repository as `kubernetes/website` (https://github.com/kubernetes/website) to avoid confusion.
I realize that `k/website` and `k8s/website` are also used as aliases or short cuts for naming the docs repository.
It's okay to suggest other changes 😄 .

**Comment by sftim** [comment]
Right, so: @MaxymVlasov could you revise just the contributor pages further.

Anywhere this PR replaces `k/website` with `kubernetes/website`, please:
- ensure it hyperlinks to https://github.com/kubernetes/website
- ensure that at least once in the same web page, we explain that contributors often shorten this to `k/website`.

**Review by MaxymVlasov — COMMENTED** [review]
@sftim This way?

**Comment by sftim** [comment]
The changes in https://github.com/kubernetes/website/pull/40493#pullrequestreview-1576751514 match what I suggested in https://github.com/kubernetes/website/pull/40493#issuecomment-1662188835
